src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -49,7 +49,7 @@ public static function register($mode = false)
             'legacy' => array(),
             'other' => array(),
         );
-        $deprecationHandler = function ($type, $msg, $file, $line, $context) use (&$deprecations, $getMode) {
+        $deprecationHandler = function ($type, $msg, $file = null, $line = null, $context = array()) use (&$deprecations, $getMode) {
```

Comment:
Fair enough, let's only change the `$context` argument here. We can challenge the decision in the future anyway in case someone can point us to an issue related to it.